_layouts/ome-community-meeting-program-2020.html

```diff
@@ -13,6 +13,7 @@ <h1>2020 OME Community Meeting</h1>
       <li class="{{ page.nav-day2 }}"><a href="{{ site.baseurl }}/events/ome-community-meeting-2020/day2/">Day 2 <span class="hide-for-small-only">- Wed May 27</span></a></li>
       <li class="{{ page.nav-workshops }}"><a href="{{ site.baseurl }}/events/ome-community-meeting-2020/workshops/">Day 3 <span class="hide-for-small-only">- Workshops</span></a></li>
       <li class="{{ page.nav-speakers }}"><a href="{{ site.baseurl }}/events/ome-community-meeting-2020/speakers/">Speakers</span></a></li>
+      <li class="{{ page.nav-speakers }}"><a href="{{ site.baseurl }}/events/ome-community-meeting-2020/lightning-talks/"><span class="show-for-small-only">Community</span> Lightning Talks</span></a></li>
```

Comment:
Here and in the line above the `<span>` tags seem unbalanced.

Reply:
Good catch, @mtbc! Fixing shortly.